crates/ruff_python_formatter/src/statement/stmt_function_def.rs

```diff
@@ -179,25 +180,3 @@ fn format_function_header(f: &mut PyFormatter, item: &StmtFunctionDef) -> Format
 
     group(&format_inner).fmt(f)
 }
-
-/// Returns `true` if [`Parameters`] is empty (no parameters, no comments, etc.).
-fn empty_parameters(parameters: &Parameters, source: &str) -> bool {
-    let mut tokenizer = SimpleTokenizer::new(source, parameters.range())
```

Comment:
I ditched the tokenizer in favor of `parameters.is_empty() && !comments.ranges().intersects(parameters.range())`.